### PR TITLE
Use conformance test artifacts

### PR DESCRIPTION
**Pull request by netdpb**
Instead of looking in the `jspecify` sibling project directory for conformance test inputs and dependencies, use the distributed artifacts produced by it.

Depends on https://github.com/jspecify/jspecify/pull/420.

Part of #107.

### CODE REVIEW

**Comment by netdpb** [comment]
This probably needs to wait until #114 is merged.

**Review by cpovirk — APPROVED** [review]

**Pull request merged**
